pkg/drivers/commonconfig.go

```diff
@@ -71,8 +71,11 @@ func GetController(ctx context.Context, cr csmv1.ContainerStorageModule, operato
 	}
 
 	controllerYAML := driverYAML.(utils.ControllerYAML)
-	controllerYAML.Deployment.Spec.Replicas = &cr.Spec.Driver.Replicas
 
+	// if using a minimal manifest, replicas may not be present.
+	if cr.Spec.Driver.Replicas != 0 {
+		controllerYAML.Deployment.Spec.Replicas = &cr.Spec.Driver.Replicas
+	}
```

Comment:
Is this really required?

Reply:
yes, because with minimal sample files, the replicas can be 0 in the CR. In this case, we want to use the default values in operatorconfig. 

Without that check, the minimal manifests would deploy the driver pods w/o the deployment pods